js_modules/dagster-ui/packages/ui-core/src/graph/layout.ts

```diff
@@ -134,7 +134,7 @@ export function layoutOpGraph(pipelineOps: ILayoutOp[], opts: LayoutOpGraphOptio
   }
 
   // Define a new top-down, left to right graph layout
-  g.setGraph({rankdir: 'TB', marginx, marginy, ranker: 'tight-tree'});
+  g.setGraph({rankdir: 'TB', marginx, marginy, ranker: 'network-simplex'});
```

Comment:
Moving op graphs back to network simplex? Not sure if this is related to the rest?

Reply:
yeah I removed that, I was testing something